### PR TITLE
Fix clean kernel command to support more versions

### PR DESCRIPTION
**Pull request by ifbizo**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -44,7 +44,7 @@ clean:
 
 .PHONY: clean_kernel
 clean_kernel:
-	rm -rf build/linux-4.*
+	rm -rf build/linux-[0-9].*
```
Comment (SolidHal) [inline makefile]: Thanks for catching that `clean_kernel` was hardcoded. 
This should probably read `rm -rf build/linux-KVER`

**Comment by ifbizo** [comment]
Yup, good catch @SolidHal about only removing a specific kernel. Fixed.

**Comment by rk-zero** [comment]
you also need to remove the downloaded linux-libre file, or the patch scripts will not be applied.

**Comment by SolidHal** [comment]
@rk-zero the kernel patches are applied if the extracted linux source isn't found, so the linux-libre-KVER-gnu.tar.lz doesn't need  to be removed. 

```
# build Linux-libre, with ath9k_htc
[ ! -f linux-libre-$KVER-gnu.tar.lz ] && wget https://www.linux-libre.fsfla.org/pub/linux-libre/releases/$KVER-gnu/linux-libre-$KVER-gnu.tar.lz
[ ! -d linux-$KVER ] && tar --lzip -xvf linux-libre-$KVER-gnu.tar.lz && FRESH=true
cd linux-$KVER
make clean
make mrproper
#Apply the usb and mmc patches if unapplied
[ "$FRESH" = true ] && for i in $RESOURCES/patches-tested/kernel/5.x-dwc2/*.patch; do echo $i; patch -p1 < $i; done
[ "$FRESH" = true ] && for i in $RESOURCES/patches-tested/DTS/5.x-dts/*.patch; do echo $i; patch -p1 < $i; done
[ "$FRESH" = true ] && for i in $RESOURCES/patches-tested/DTS/*.patch; do echo $i; patch -p1 < $i; done
[ "$FRESH" = true ] && for i in $RESOURCES/patches-tested/kernel/*.patch; do echo $i; patch -p1 < $i; done

```

**Pull request merged**